src/components/CardAdding/CardAdding.js

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+import { v4 } from "uuid";
+
+import "./CardAdding.css";
+
+function CardAdding(props) {
+  const addCardButtonText = "OK",
+    cancelButtonText = "Cancel",
+    cardTitle = "Title...",
+    cardText = "Text...";
+  const [newCardData, setnewCardData] = useState({
+    enteredTitle: "",
+    enteredText: "",
+  });
+
+  const newCardTitleHandler = (event) => {
+    setnewCardData((prevState) => {
+      return { ...prevState, enteredTitle: event.target.value };
+    });
+  };
+
+  const newCardTextHandler = (event) => {
+    setnewCardData((prevState) => {
+      return { ...prevState, enteredText: event.target.value };
+    });
+  };
+
+  const onCardAddingClick = () => {
+    props.onAddCard({
+      id: v4(),
+      title: newCardData.enteredTitle,
+      text: newCardData.enteredText,
+      check: false,
+    });
+    props.addCardMode(false);
```

Comment:
You manipulate parent's state from the child component.
Isn't it better to control the state of component within the component itself?
What do you think?